Formula/gdk-pixbuf.rb

```diff
@@ -13,6 +13,10 @@ class GdkPixbuf < Formula
 
   option :universal
   option "with-relocations", "Build with relocation support for bundles"
+  option "without-modules", "Disable dynamic module loading"
+  option "with-included-loaders=", "Build the specified loaders into gdk-pixbuf"
+
+  deprecated_option "disable-modules" => "without-modules"
```

Comment:
Given the underlying configure flag is named `--disable-modules`, would the expectation be to map it to `--without-modules` or expose it as-is?